package.json

```diff
@@ -23,7 +27,8 @@
     "lint": "eslint ./src/**/*.js -f stylish --fix",
     "storybook": "start-storybook -p 6006 -s public",
     "build-storybook": "build-storybook -s public",
-    "prettier": "prettier -w './src/**/*.{js,jsx}'"
+    "prettier": "prettier -w './src/**/*.{js,jsx}'",
+    "rustywind": "rustywind --write ."
```

Comment:
To simplify dev flow instead of running 2 commands, you can simply use post hooks ( More info here https://docs.npmjs.com/cli/v8/using-npm/scripts#pre--post-scripts ), so after running prettier rustywind will be executed automatically